### PR TITLE
Fix/gravity input

### PR DESCRIPTION
**Pull request by alexthompson207**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [X] Fix
- [ ] Feature
- [ ] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Add a function to disable `e, -, E, +, . ` keyDown events.
Add a function to disable weight input for numbers larger than 9999999.
Adjust the width of input to match 9 characters

### What ticket(s) do the changes resolve?
#65 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
None.

### How to Test Changes?
A user should not be able to input `e, -, E, +,  . `  keys or type a number larger than 9999999.

### CODE REVIEW

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 1]
```diff
@@ -23,6 +23,20 @@ const PlanetInfo: React.FC<InfoProps> = ({ currentPlanet, resetSort }): JSX.Elem
...
 
+  const handleInput = (e: React.KeyboardEvent<HTMLInputElement>): void => {
+    if (e.key === "e" || e.key === "." || e.key === "+" || e.key === "-" || e.key === "E") {
+      e.preventDefault();
+    }
+  }
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
+    if (Number(event.target.value) > 9999999) {
+      event.preventDefault();
```
Comment (knees4bees) [inline src/components/PlanetInfo/PlanetInfo.tsx:34]: Yes yes yes this is such a simple solution that does exactly what we want!

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 2]
```diff
@@ -23,6 +23,20 @@ const PlanetInfo: React.FC<InfoProps> = ({ currentPlanet, resetSort }): JSX.Elem
     return weightOnPlanet;
   }
 
+  const handleInput = (e: React.KeyboardEvent<HTMLInputElement>): void => {
+    if (e.key === "e" || e.key === "." || e.key === "+" || e.key === "-" || e.key === "E") {
```
Comment (knees4bees) [inline src/components/PlanetInfo/PlanetInfo.tsx:27]: Excellent!

**Review by knees4bees — APPROVED** [review]
Well done, Alex.

**Review by ConnorAndersonLarson — APPROVED** [review]
Nice! Can't let those kids break our stuff!

**Pull request merged**